src/diff/json.js

```diff
@@ -1,8 +1,9 @@
 import Diff from './base';
 import {lineDiff} from './line';
 
-const objectPrototypeToString = Object.prototype.toString;
-
+const objectPrototype = Object.prototype;
+const objectPrototypeToString = objectPrototype.toString;
+const objectPrototypeHasOwnProperty = objectPrototype.hasOwnProperty;
```

Comment:
Stylistic opinion: these variables (including the existing `objectPrototypeToString` one that was there before your PR) are pointless indirection and it'd be clearer and nicer to just do `Object.prototype.hasOwnProperty.call(...)` instead of `objectPrototypeHasOwnProperty.call(...)`.

Reply:
(I went ahead and tweaked this.)